Doc/library/stdtypes.rst

```diff
@@ -5394,27 +5427,6 @@ information.  There is exactly one ``NotImplemented`` object.
 It is written as ``NotImplemented``.
 
 
-.. _bltin-boolean-values:
```

Comment:
Hello. The removal of this `ref` tag broke intersphinx downstream. I have not tracked down yet which middleware is calling this tag when API docstring has a "bool" mentioned. I suspect it is numpydoc. Is it not possible to reuse this tag for your new section above?

xref https://github.com/astropy/astropy/issues/15428

Reply:
If you open a PR, I can backport it to the 3.12 branch

Reply:
Sure. Please see https://github.com/python/cpython/pull/110371 . Thank you for your consideration!